vimeo-networking/src/main/java/com/vimeo/networking/model/Video.java

```diff
@@ -102,17 +105,25 @@ public String toString() {
     public Date modifiedTime;
     public ArrayList<String> contentRating;
     public String license;
-    public com.vimeo.networking.model.Privacy privacy;
+    public Privacy privacy;
     public PictureCollection pictures;
     public ArrayList<Tag> tags;
     public StatsCollection stats;
     public Metadata metadata;
-    public com.vimeo.networking.model.User user;
+    public User user;
     private Status status;
     public VideoLog log;
     public List<Category> categories;
 
+    /**
+     * This getter is always guaranteed to return a {@link Status}, {@link Status#NONE if null}. If the Status
+     * is equal to {@link Status#TRANSCODE_STARTING} then we'll just return the Status {@link Status#TRANSCODING}
+     * since they're functionally equivalent from a client-side perspective.
+     */
     public Status getStatus() {
+        if (status == Status.TRANSCODE_STARTING) {
+            return Status.TRANSCODING;
+        }
```

Comment:
The way I look at it, a vast majority of the use cases we're not going to want to worry about this state that only lasts for a "very temporary" amount of time. And it would only complicate the consumption of our client side API to cause other consumers of the library to have to worry about this additional status when they most likely also won't care.

I don't think it's necessarily dishonest because `TRANSCODE_STARTING` is technically just a phase of `TRANSCODING`.

That being said, it's included in the API request for what I can only image is for technical transparency. So possibly a solution that can cater to both parties would be an additional getter which returns the raw value. And this one can be renamed to `getSimpleStatus()`. Or the raw one can be named `getAdvancedStatus()` or `getRawStatus()` since it caters to the (presumed) minority. We could also use `getRawStatus()` for any logging which will give us that technical transparency.